test/WebJobs.Extensions.Tests/Extensions/DocumentDB/DocumentDBConfigurationTests.cs

```diff
@@ -86,6 +118,33 @@ public void CreateContext_UsesDefaultRetryValue()
             Assert.Equal(DocumentDBContext.DefaultMaxThrottleRetries, context.MaxThrottleRetries);
         }
 
+        [Theory]
+        [InlineData(typeof(IEnumerable<string>), true)]
+        [InlineData(typeof(IEnumerable<Document>), true)]
+        [InlineData(typeof(IEnumerable<JObject>), true)]
+        [InlineData(typeof(JArray), false)]
+        [InlineData(typeof(string), false)]
+        [InlineData(typeof(List<Document>), false)]
```

Comment:
could make the argument we want to support types that implement IEnumerable but it's probably better to keep it simple